tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -251,20 +251,15 @@
     <!-- Latest package https://www.nuget.org/packages/System.Net.Requests/4.3.0 -->
     <PackageReference Include="System.Net.Requests" Version="4.3.0" />
 
-    <!-- Integration: Microsoft.AspNetCore.Html.Abstractions -->
-    <!-- Assembly: Microsoft.AspNetCore.Html.Abstractions -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.Html.Abstractions/2.2.0 -->
-    <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="2.2.0" />
-
     <!-- Integration: xunit.execution.desktop -->
     <!-- Assembly: xunit.execution.desktop -->
-    <!-- Latest package https://www.nuget.org/packages/xunit/2.6.6 -->
+    <!-- Latest package https://www.nuget.org/packages/xunit/2.7.0 -->
     <PackageReference Include="xunit" Version="2.6.6" />
```

Comment:
I excluded the xunit update, because it's not currently working (likely requires a test SDK update)